src/components/Application/PageElements.tsx

```diff
@@ -86,13 +95,88 @@ const PageElements: React.FC<PageElementProps> = ({
   if (isReview)
     return (
       <Form>
-        {elements.map((element) => {
-          return <p>Review Element</p>
-        })}
+        <Segment.Group>
+          {elements.map(({ element, thisReviewLatestResponse }) => (
+            <Segment key={`question_${element.id}`}>
+              <Grid columns="equal">
+                <Grid.Column floated="left">
+                  <SummaryViewWrapperNEW {...getSummaryViewProps(element)} />
+                </Grid.Column>
+                <Grid.Column floated="right" textAlign="right">
+                  <ReviewButton
```

Comment:
Maybe this one should be renamed to `ApproveAllButton`?

Reply:
5221d0610db840c66350011d38ea137b82178e3e